pages/2017/index.rst

```diff
@@ -8,4 +8,60 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
+
+As with the two previous Python in Astronomy conferences we aim to bring a wide
```

Comment:
Maybe "As with the two previous Python in Astronomy conferences we aim" -> "This conference aims" or similar?  I think it's better to just say the goals than make it sound like we're stuck in the past ;)